lib/htmltoword/document.rb

```diff
@@ -61,10 +66,21 @@ def generate
               out.write(source)
             elsif @replaceable_files[entry.name]
               out.write(@replaceable_files[entry.name])
+            elsif entry.name == Document.content_types_xml_file
```

Comment:
Not a big fan of this, specially if it might affect the custom templates users might use (affecting a PartName or something). What about maybe using a transformation to add the ContentTypes that are actually in use?